spec/features/job_seekers_can_view_vacancies_spec.rb

```diff
@@ -125,6 +125,28 @@
     expect(page).not_to have_content(I18n.t('jobs.no_jobs'))
   end
 
+  scenario 'Should have button with text \'Search\' if default search criteria are applied' do
+    Vacancy.__elasticsearch__.client.indices.flush
+    visit jobs_path
+
+    # expect(page).to have_content(I18n.t('buttons.apply_filters'))
+    expect(find('.button').value).to eq(I18n.t('buttons.apply_filters'))
+  end
```

Comment:
I wonder if we do need this scenario at all since the other scenario caters for both states. Maybe rewording the title eg `The search button text changes from 'Search' to 'Refine search' when the filters are applied`.

What do you think?

Reply:
Agreed 👍